src/libraries/System.Text.Json/tests/Serialization/Object.WriteTests.cs

```diff
@@ -80,8 +80,8 @@ private class Indexer
         [Fact]
         public static void WriteObjectWorks_ReferenceTypeMissingPublicParameterlessConstructor()
         {
-            PublicParameterizedConstructorTestClass paramterless = PublicParameterizedConstructorTestClass.Instance;
-            Assert.Equal("{\"Name\":\"42\"}", JsonSerializer.Serialize(paramterless));
+            PublicParameterizedConstructorTestClass parameterless = PublicParameterizedConstructorTestClass.Instance;
```

Comment:
How did you detect these or did you happen to just run across them ad-hoc?

Reply:
Funny story - my tests were broken in https://github.com/dotnet/runtime/pull/45911, and I noticed that the exception message coming from mono had this spelling issue. So I grepped the whole repo for the same spelling error and this came up.

Then my eye caught the "Polymorphic" spelling mistake below, so I figured I had to fix that as well, while I was in here.

Reply:
Ah, good old grepping for a string with a typo :)